src/Microsoft.Health.Fhir.Api/Features/Filters/ValidateExportRequestFilterAttribute.cs

```diff
@@ -32,6 +32,8 @@ public ValidateExportRequestFilterAttribute()
             _supportedQueryParams = new HashSet<string>(StringComparer.Ordinal)
             {
                 KnownQueryParameterNames.Since,
+                KnownQueryParameterNames.AnonymizationConfigurationLocation,
+                KnownQueryParameterNames.AnonymizationConfigurationFileEtag,
```

Comment:
These query params are only supported for the base $export endpoint, not the patient or group endpoints.

Reply:
I think having a special case for the default export would be good as a user could enter these parameters for the other export endpoints and get no indication that they were not honored. You should be able to use context.HttpContext.Request.Path to figure out which endpoint the request is being sent to.

Reply:
Thanks, updated.